test/spec/features/completer/completer_spec.rb

```diff
@@ -51,14 +51,19 @@
 
   context 'master queries' do
     context 'with current master set' do
-      it 'can complete node names' do
+      it 'can complete grid names' do
+        node_names = run('kontena node ls -q').out.split(/[\r\n]+/)
```

Comment:
yes, looks like tty-table adds trailing space to single column output. e2e finding bugs like crazy.